src/nvim/keymap.c

```diff
@@ -582,7 +581,7 @@ int find_special_key(const char_u **srcp, const size_t src_len, int *const modp,
         // Anything accepted, like <C-?>.
         // <C-"> or <M-"> are not special in strings as " is
         // the string delimiter. With a backslash it works: <M-\">
-        if (end - bp > l && !(in_string && bp[1] == '"') && bp[2] == '>') {
+        if (end - bp > l && !(in_string && bp[1] == '"') && bp[l+1] == '>') {
```

Comment:
I wonder why cb02137dfac7357650a2e9cc32acb66326e59058 changed it to `bp[2]`. @jamessan what do you think?

Reply:
Because that's what upstream did (and still does).  Cc @brammool and @chrisbra.